Compiling 1.7.0 fails

I originally filed an issue where 1.7.0 did not compile, but I had to add -D__ANDROID_NDK__ to the CFLAGS in my Android.mk, 
 probably because I'm using an older NDK (r20b).

But now I encounter something that is not NDK-related:

int32_t getMDelayBeforeCloseMillis() const is defined but not implemented
int32_t getDelayBeforeCloseMillis() is implemented in AudioStream.h however and AS says it is never used.

And several warnings like:

/oboe/src\aaudio/AudioStreamAAudio.h:123:44: warning: parameter 'mDelayBeforeCloseMillis' shadows member inherited from type 'AudioStream' [-Wshadow-field]
    void setDelayBeforeCloseMillis(int32_t mDelayBeforeCloseMillis);
                                           ^
oboe/include\oboe/AudioStream.h:597:26: note: declared here
    int32_t              mDelayBeforeCloseMillis = kMinDelayBeforeCloseMillis;


Seems this method should be changed to:
void setDelayBeforeCloseMillis(int32_t delayBeforeCloseMillis) as in its implementation

Actually, these should just be removed from AudioStreamAAudio.h:
    int32_t getDelayBeforeCloseMillis() const;

    void setDelayBeforeCloseMillis(int32_t delayBeforeCloseMillis);

Then it's all fine.

--- a/src/aaudio/AudioStreamAAudio.h
+++ b/src/aaudio/AudioStreamAAudio.h
@@ -117,11 +117,6 @@ private:
      */
     void launchStopThread();
 
-public:
-    int32_t getMDelayBeforeCloseMillis() const;
-
-    void setDelayBeforeCloseMillis(int32_t mDelayBeforeCloseMillis);
-
 private:
 
     std::atomic<bool>    mCallbackThreadEnabled;

